src/redux/actions/index.js

```diff
@@ -0,0 +1,33 @@
+import {
+  FETCH_DEFAULT,
+  FETCH_FROM_SEARCH,
+  CHANGE_SEARCH,
+  CHANGE_SORTING
+} from '../constants/action-types';
+import { fetchFromSearch, fetchDefault } from '../../utils/dataLoaders';
+
+export function changeSearch(text) {
+  return { type: CHANGE_SEARCH, payload: text };
+}
+
+export function changeSorting(text) {
+  return { type: CHANGE_SORTING, payload: text };
+}
+
+export function getDefaultData(limit) {
```

Comment:
Currently this two action creators creates function to start loading movies, and then dispatch action to put them into our store. There are 3 things to think about:
1. Request can take some time. It also can fail due to any reason. For better UX we can notify user about this stages. There is a common practice to have 3 states for async actions: loading state while request in progress, resolved state when request completed and invalid state when request failed (you can use something like START_LOADING, STOP_LOADING, FETCH_MOVIES_COMPLETE, FETCH_ERROR as action types for that).
2. Both actions do the same - load movies, but with different arguments. That's why they have equal reducers. You can dispatch the same action in both cases or reduce swtich-case operator.
3. As a user i expect to see my previous search result when page reloaded (because of persistence enabled), but i always see this default request. So i think, we can remove it.